shortfin/tests/apps/llm/components/scheduler_test.py

```diff
@@ -137,7 +137,7 @@ def test_scheduler_reserved_extra():
     scheduler = Scheduler(ideal_batch_size=ideal_batch_size)
 
     reserve_helper(scheduler, rid=0, count=5)
-    workload = make_workload({0: 5, 1: 3})
```

Comment:
We shouldnot be changing the scheduler tests.

Reply:
@rsuderman this change reverts the changes in scheduler tests back to the original.